src/bandersnatch/mirror.py

```diff
@@ -211,7 +211,6 @@ def __init__(
         self.homedir = self.storage_backend.PATH_BACKEND(str(homedir))
         self.lockfile_path = self.homedir / ".lock"
         self.master = master
-        self.filters = LoadedFilters(load_all=True)
```

Comment:
This seems a big change. Why? Can you explain how the functionality has been retained from removing this please or how it's was not needed.

Reply:
I should probably not have done this as part of an enhancement, but it's a superflous line. It's set in the superclass's constructor.